pf-backupandclean.cmd

```diff
@@ -0,0 +1,9 @@
+if %BUILD_TOOLS% EQU "" goto EOF
+
+set SCRIPT_PATH=%~dp0
+set PF_ROOT=%SCRIPT_PATH%..\..\..
+
+set PFSDK_BACKUP_VER=%DATE:~10,4%-%DATE:~7,2%-%DATE:~4,2%.%TIME:~0,2%-%TIME:~3,2%-%TIME:~6,2%
+%BUILD_TOOLS%\vZip.exe /FOLDER:%PF_ROOT%\sdks\XPlatCSdk /OUTPUTNAME:%PF_ROOT%\sdks\XPlatCSdk-%PFSDK_BACKUP_VER%.zip /EXCLUDE:external,.git,.vs,x64
```

Comment:
Where are build tools coming from? might be useful to include a usage message for this script to clarify what its doing/how it should be used

Reply:
they are from sdk.buildtools enlistment.  you'll have to define this var that outside this script.  i can put a comment in

---
In reply to: [631947770](https://github.com/jplafonta/XPlatCTemplate/pull/16#discussion_r631947770) [](ancestors = 631947770)